src/styles/about.scss

```
@@ -0,0 +1,60 @@
+.about {
+  padding-top: 104px;
+  display: flex;
+
+  @include media-flex($smallWidth);
+
+  &__heading {
+    width: 50%;
+    margin: 0;
+    font-weight: bold;
+    font-size: 48px;
+    @include content-width-small-screen;
+
+    @media (max-width: 600px) {
+      & {
+        font-size: 36px;
+      }
+    }
+
+    &--name {
+      display: block;
+    }
+
+    &--link {
+      font-family: $robotoFontFamily;
+      text-decoration: none;
+      font-size: 48px;
+      color: $blueColor;
+      border-bottom: 4px solid transparent;
+      transition: border-bottom 0.5s;

```

create variable for the duration
```suggestion
      transition: border-bottom $duration;

```

---

окей